contrib/hbmxml/tests/tohash.prg

```diff
@@ -52,9 +53,18 @@ STATIC FUNCTION s_NodeToHash( pNode )
                   ENDIF
                   AAdd( hHash[ mxmlGetElement( pNode ) ], hHashChild )
                ENDIF
-            ELSEIF hb_mxmlGetAttrsCount( pNode ) > 0
-               hHash[ mxmlGetElement( pNode ) + "@attr" ] := hb_mxmlGetAttrs( pNode )
+            ELSE
+               IF HB_MXMLGETATTRSCOUNT( pNode ) > 0
```

Comment:
Keep the original `ELSEIF` condition here, the update just breaks formatting for no reason.